### ISSUE TITLE
QoL request

### ISSUE THREAD

**Issue by GhostHawk272** [zjosua/anki-mc#134]
Is it possible to set the default question type to a specific number? I only use option 2 (sc), so I was wondering whether it would be possible to automatically set this as a default when I select the card - as this would save me a lot of time from having to click and type in 2 then click the next box. Ok, this only takes like 5 seconds to do, but with hundreds of thousands of cards, it accumulates to a decent chunk of time saved. Many thanks!

**Comment by Schlauer-Hax** [comment]
I don't think that this is possible, because Anki has no option to set a default value on fields. Greetings
